### PR TITLE
Remove deprecated `method` boot option

### PR DESCRIPTION
**Pull request by KKoukiou**
This is deprecated ~10 years already.

https://issues.redhat.com/browse/INSTALLER-1818

### CODE REVIEW

File: pyanaconda/payload/dnf/payload.py [hunk 1]
```diff
@@ -89,9 +89,9 @@ def _set_default_source(self, opts):
         if self.proxy.Sources:
             log.debug("The DNF source is already set.")
 
-        elif opts.method:
+        elif opts.repo:
             log.debug("Use the DNF source from opts.")
-            source_proxy = self._create_source_from_url(opts.method)
```
Comment (jkonecny12) [inline pyanaconda/payload/dnf/payload.py:94]: Hmm, I wonder about this. We should run payload kickstart tests to test different payloads are correctly created.

**Review by jkonecny12 — APPROVED** [review]
Looks good to me. Let's just make sure that we don't break anything.

File: pyanaconda/payload/dnf/payload.py (hunk 1 above)
Comment (KKoukiou) [reply pyanaconda/payload/dnf/payload.py:94]: I also found it weird.

**Pull request merged**